rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/search.vectors/90_sparse_vector.yml

```diff
@@ -97,6 +121,26 @@
   - match: { hits.hits.0._id: "2" }
   - match: { hits.hits.1._id: "1" }
 
+  - do:
```

Comment:
Your code will be introduced into version 8.12, so all these tests will probably need to be within their own clause as calls against 8.11 will result in these tests failing because in 8.11, sparse_vector mapping was added back, but exists queries were not yet supported.

Reply:
NOTE: My BWC testing concern here only applies if this change doesn't make it into 8.11 :)

Reply:
Keeping that as a note, and will update depending on whether it makes the cut for 8.11 :)